README.md

```diff
@@ -79,16 +79,16 @@ With `uno` script set up, you can now use it to download, configure, and run Flu
 
 ## Fetch command
 
-The `uno fetch` command fetches the binary tarball dependencies of Apache Fluo to be later installed
-by the `setup` command. By default, it will download binary tarballs. However, you can configure the
-`fetch` command to build Fluo or Accumulo from a local git repo by setting `FLUO_REPO` or
+The `uno fetch <component>` command fetches the binary tarballs of a component and its dependencies to
```

Comment:
I would write this as follows.. but I am ok with leaving it as is.

The `uno fetch <component>` command fetches the tarballs of a component and its dependencies for later use by the `setup` command